### PR TITLE
Lektor-friendly deployment

### LINKED ISSUE #5: Fix dockerfile to build with lektor
- [x] install/build using lektor in the dockerfile
- [x] remove index.html

### PR DESCRIPTION
**Pull request by tiborsimko**
* Amends building and deployment procedures to use Lektor. (closes #5)

Signed-off-by: Tibor Simko <tibor.simko@cern.ch>

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 FROM nginx:1.13
 EXPOSE 8080
-ADD index.html /usr/share/nginx/html/
+ADD _build /usr/share/nginx/html/
```
Comment (jbenito3) [inline Dockerfile]: I would suggest using COPY instead of ADD (to avoid issues in the future due to ADD magic)

File: Dockerfile [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
 ADD nginx.conf /etc/nginx/conf.d/default.conf
```
Comment (jbenito3) [inline Dockerfile]: same here (see above)

File: README.rst [hunk 3]
```diff
@@ -12,18 +12,36 @@ computational research data analysis workflows on remote containerised compute
...
+This repository holds the REANA project web site. It uses `Lektor
+<https://www.getlektor.com/>`_ static web site generator.
 
 Installation
 ------------
 
 .. code-block:: console
 
+   $ mkvirtualenv reana.io
+   $ pip install -r requirements.txt
```
Comment (jbenito3) [inline README.rst]: I guess you have a reason not to install lektor (and build) with docker?

Comment (tiborsimko) [reply README.rst]: Didn't want to add Python and its dependencies into the Nginx image, but we can do that indeed... Better for reproducibility :wink:

**Pull request merged**